cmd/kinflate/README.md

```diff
@@ -1,12 +1,8 @@
-# kinflate
-
-`kinflate` is a kubernetes cluster configuration
-utility.
-
-It reads a cluster app definition from a
-directory of YAML files, and creates new YAML suitable
-for directly piping to `kubectl apply`.
+# Overview
 
+`kinflate` is a command line interface to manage Kubernetes Application 
+configuration.  It reads an app definition from a directory of YAML files, and 
+creates new YAML suitable for directly piping to `kubectl apply`.
 
 ## Declarative Application Management
```

Comment:
this paragraph is not useful to anyone, do not expect users to read the proposal or even get familiar with the acronym. Kubernetes users already know that YAML files are declarative application management.